eviatar/enableFeatures_createUsers.ps1

```diff
@@ -0,0 +1,54 @@
+﻿function Install-WindowsFeatures {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string[]]$Features
+    )
+    
+    foreach ($Feature in $Features) {
+        Write-Host "Installing feature: $Feature"
+        Install-WindowsFeature -Name $Feature -IncludeAllSubFeature -IncludeManagementTools -Verbose
+    }
+}
+
+function Create-LocalUsers {

```

Comment:
Create is not a valid PowerShell verb... are you sure this is the best name for your function?

Reply:
The function should be renamed to New-LocalUsers to align with best practices.